pandas/core/categorical.py

```diff
@@ -602,6 +602,21 @@ def _get_categories(self):
     categories = property(fget=_get_categories, fset=_set_categories,
                           doc=_categories_doc)
 
+    def _codes_for_groupby(self, sort=False):
+        """ Return a Categorical adjusted for groupby """
```

Comment:
if you can add a doc-string (Parameters, Returns), just like its a publick method.